[2025-02-06T08:32:53.929+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-06T08:32:53.947+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data scheduled__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-06T08:32:53.956+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_que_faire_a_paris.extract_data scheduled__2025-02-05T00:00:00+00:00 [queued]>
[2025-02-06T08:32:53.956+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-06T08:32:53.970+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): extract_data> on 2025-02-05 00:00:00+00:00
[2025-02-06T08:32:53.978+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3529) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-06T08:32:53.980+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_que_faire_a_paris', 'extract_data', 'scheduled__2025-02-05T00:00:00+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/dag_que_faire_a_paris.py', '--cfg-path', '/tmp/tmpw9pd1pty']
[2025-02-06T08:32:53.982+0000] {standard_task_runner.py:72} INFO - Started process 3531 to run task
[2025-02-06T08:32:53.984+0000] {standard_task_runner.py:105} INFO - Job 96: Subtask extract_data
[2025-02-06T08:32:54.023+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_que_faire_a_paris.extract_data scheduled__2025-02-05T00:00:00+00:00 [running]> on host b176106a6d6f
[2025-02-06T08:32:54.384+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_que_faire_a_paris' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-05T00:00:00+00:00'
[2025-02-06T08:32:54.388+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-06T08:32:57.589+0000] {dag_que_faire_a_paris.py:40} INFO - âœ… Extraction completed successfully!
[2025-02-06T08:32:57.591+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-06T08:32:57.597+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-06T08:32:57.597+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_que_faire_a_paris, task_id=extract_data, run_id=scheduled__2025-02-05T00:00:00+00:00, execution_date=20250205T000000, start_date=20250206T083253, end_date=20250206T083257
[2025-02-06T08:32:57.614+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-06T08:32:57.627+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-06T08:32:57.630+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
